pset/updater_test.go

```diff
@@ -77,3 +85,132 @@ func TestUpdaterAddOutput(t *testing.T) {
 	assert.Equal(t, 1, len(updater.Upsbt.UnsignedTx.Outputs))
 	assert.Equal(t, 1, len(updater.Upsbt.Outputs))
 }
+
+func TestCreateAddIssuanceSignAndBroadcast(t *testing.T) {
```

Comment:
I do believe that this  e2e test belongs o pset_test rather than here.

Reply:
Done